src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,86 @@
+package lotto.domain;
+
+import lotto.exception.LottoTicketException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+    private static final int SUBLIST_FROM_INDEX = 0;
+    private static final int SUBLIST_TO_INDEX = 6;
+    private static final int LOTTO_SIZE = 6;
+    private static final String DELIMITER = ",";
+    private static final int COUNT_FOR_SECOND_RANK = 5;
+
+    private final List<LottoNumber> numbers;
+
+    private LottoTicket(List<LottoNumber> numbers) {
+        this.numbers = numbers;
```

Comment:
문제가 될만한 모든 유효성 검사는 생성자에서 해야 합니다. 
자동, 수동을 LottoTicket에서 생성하는 현재 구조에서는 문제가 없겠지만.. 나중에 자동, 수동 역할을 분리한다면 다음과 같은 문제가 생길 수 있습니다. 

```java
    @Test
    void name() {
        List<LottoNumber> list = new ArrayList<>(Arrays.asList(
                LottoNumber.valueOf(1),
                LottoNumber.valueOf(2),
                LottoNumber.valueOf(3),
                LottoNumber.valueOf(4),
                LottoNumber.valueOf(5),
                LottoNumber.valueOf(6)));

        LottoTicket lottoTicket = new LottoTicket(list);

        list.add(LottoNumber.valueOf(10));

        assertThat(list).isEqualTo(7);
        assertThat(lottoTicket.getNumbers()).isEqualTo(6);
    }
```

테스트의 결과는 어떨까요?

Reply:
앗, 외부에서 내부 필드값을 조작할 수 있는 여지가 남아있네요... 생성자에서 
```suggestion
            private LottoTicket(List<LottoNumber> numbers) {
        this.numbers = new ArrayList<>(numbers);

        validateNumbersCount(numbers);
        validateNumbersDuplication(numbers);
    }
```
이같은 방법으로 불변을 유지할 수 있을 것 같습니다!

혹시 "문제가 될만한 모든 유효성 검사는 생성자에서 해야 합니다."라는 피드백이 이같은 맥락에서 말씀해주신 걸까요? 😄